qp_qiime2/tests/test_qiime2.py

```diff
@@ -224,9 +228,112 @@ def test_beta_phylogenetic(self):
                  'distance-matrix.tsv'), 'plain_text')])
         self.assertEqual(ainfo[0].output_name, 'distance_matrix')
 
+    def test_beta_correlation(self):
+        # as we don't have a distance matrix, we will process one first
+        params = {
+            'A pseudocount to handle zeros for compositional metrics.  This '
+            'is ignored for other metrics.': '1',
+            'The beta diversity metric to be '
+            'computed.': "Rogers-Tanimoto distance",
+            'The feature table containing the samples over which beta '
+            'diversity should be computed.': '8',
+            'The number of jobs to use for the computation. This works '
+            'by breaking down the pairwise matrix into n_jobs even slices '
+            'and computing them in parallel. If -1 all CPUs are used. If '
+            '1 is given, no parallel computing code is used at all, which '
+            'is useful for debugging. For n_jobs below -1, (n_cpus + 1 + '
+            'n_jobs) are used. Thus for n_jobs = -2, all CPUs but one are '
+            'used. (Description from sklearn.metrics.pairwise_distances)': '1',
+            'qp-hide-method': 'beta',
+            'qp-hide-paramThe beta diversity metric to be computed.': 'metric',
```

Comment:
Just to confirm, is this okay, with no spaces or ' etc?
_'qp-hide-paramThe beta diversity..._